configure.ac

```diff
@@ -507,6 +509,12 @@ else
 fi
 CFLAGS_ADD="${CFLAGS_ADD} -pthread"
 
+dnl The current supported standard is C99 with gnu extensions
+
+if test "x$force_gnu99" = "xyes"; then
+	CFLAGS_ADD="${CFLAGS_ADD} --std=gnu99"
```

Comment:
This should be `-std` instead of `--std`

Reply:
thanks, done.

Reply:
Well, it does not really matter in case of gcc:
https://github.com/gcc-mirror/gcc/blob/74157df9c26d1f053678d76e60d065a67815660f/gcc/opts-common.c#L456-L457

But only `-std` is documented. I don't know about clang, it should work as well.

Reply:
Thanks!